server/src/pages/homepage/view.js

```diff
@@ -601,45 +617,66 @@ class Body extends React.Component {
             <div className="container">
               <div className="banner-content">
                 <h1>Firefox Screenshots</h1>
-                <p>Screenshots made simple. Take, save, and share screenshots without leaving Firefox.</p>
+                <Localized id="gScreenshotsDescription">
+                  <p>Screenshots made simple. Take, save, and share screenshots without leaving Firefox.</p>
+                </Localized>
                 { this.renderGetFirefox() }
               </div>
             </div>
           </div>
         </header>
-        <h2>How Screenshots Works</h2>
+        <Localized id="homePageHowScreenshotsWorks">
+          <h2>How Screenshots Works</h2>
+        </Localized>
         <section id="section-1">
           <div className="container">
             <div className="section-content">
-              <h3>Get <span className="bold">Started</span></h3>
-              <p>Find the new Screenshots icon on your toolbar. Select it, and the Screenshots menu will appear on top of your browser window.</p>
+              <Localized id="homePageGet">
+                <h3>Get <Localized id="homePageStarted"><span className="bold">Started</span></Localized></h3>
+              </Localized>
+              <Localized id="homePageGetStartedDescription">
+                <p>Find the new Screenshots icon on your toolbar. Select it, and the Screenshots menu will appear on top of your browser window.</p>
+              </Localized>
             </div>
             <div className="section-image"></div>
           </div>
         </section>
         <section id="section-2">
           <div className="container">
             <div className="section-content">
-              <h3><span className="bold">Capture</span> a Region</h3>
-              <p>Click and drag to select the area you want to capture. Or just hover and click — Screenshots will select the area for you. Like what you see? Select Save to access your screenshot online or the down arrow button to download it to your computer.</p>
+              <Localized id="homePageCaptureRegion">
+                <h3><Localized id="homePageCapture"><span className="bold">Capture</span></Localized> a Region</h3>
```

Comment:
Hmm. This example seems to suggest passing in nested components as variables, so I'll switch to that https://github.com/projectfluent/fluent.js/wiki/Localized